kubernetes/platform-services/values.yaml

```diff
@@ -26,69 +26,86 @@ configServicePort: &configServicePort 9012
 gatewayServiceHost: &gatewayServiceHost hypertrace-data-query-service
 gatewayServicePort: &gatewayServicePort 9001
 
-hypertrace-oc-collector:
+hypertrace-collector:
   resources:
-    requests:
-      cpu: "0.5"
-      memory: "256Mi"
     limits:
-      cpu: "1"
-      memory: "256Mi"
-  containerPorts:
-    - name: grpc-opencensus
-      containerPort: 55678
-    - name: http-jaeger
-      containerPort: 14268
-    - name: jaeger-tchannel
-      containerPort: 14267
-    - name: zipkin
-      containerPort: 9411
+      cpu: 1
+      memory: 2Gi
+    requests:
+      cpu: 200m
+      memory: 400Mi
   service:
     type: LoadBalancer
     ports:
-      - name: grpc-opencensus
-        port: 55678
-        targetPort: 55678
-        protocol: TCP
-      - name: http-jaeger
-        port: 14268
-        targetPort: 14268
-        protocol: TCP
-      - name: jaeger-tchannel
-        port: 14267
-        targetPort: 14267
-        protocol: TCP
-      - name: zipkin
-        port: 9411
-        targetPort: 9411
-        protocol: TCP
+    - name: grpc-otlp
+      port: 4317
+      targetPort: 4317
+      protocol: TCP
+    - name: http-otlp
+      port: 55681
+      targetPort: 55681
+      protocol: TCP
+    - name: grpc-opencensus
+      port: 55678
+      targetPort: 55678
+      protocol: TCP
+    - name: http-jaeger
+      port: 14268
+      targetPort: 14268
+      protocol: TCP
+    - name: grpc-jaeger
+      port: 14250
+      targetPort: 14268
+      protocol: TCP
+    - name: http-zipkin
+      port: 9411
+      targetPort: 9411
+      protocol: TCP
   configMap:
-    name: hypertrace-oc-collector-conf
     data:
-      log-level: INFO
+      extensions:
+        health_check: {}
       receivers:
-        opencensus:
-          # keepalive settings can help load balancing, see receiver/README.md for more info.
-          keepalive:
-            server-parameters:
-              max-connection-age: 120s
-              max-connection-age-grace: 30s
-        jaeger: {}
-        zipkin: {}
+        otlp:
```

Comment:
The only thing we are the overriding receiver and exporter property
- https://github.com/hypertrace/hypertrace-collector/blob/main/helm/values.yaml
-